README.md

```diff
@@ -39,8 +39,13 @@ If you update Vagrant, you may need to update your vagrant plugins with `vagrant
 4. Log in to the virtual machine (the VM): `vagrant ssh`
 5. Build, install, and enable demo content: `phing build install-all`
 6. Build the `/src` directory and symlink modules there to make development easier: `phing init`
-7. Visit your D8 site at [http://fs-demo.d8.local](http://fs-demo.d8.local)
-8. Visit your D7 site at [http://fs-demo.d7.local](http://fs-demo.d7.local)
+7. Visit your D8 (standalone) site at [http://fs-demo.d8.local](http://fs-demo.d8.local)
+8. Visit your D8 (domain access) site at:
+   - [http://fs-demo.d8-1.local](http://fs-demo.d8-1.local)
+   - [http://fs-demo.d8-2.local](http://fs-demo.d8-2.local)
+   - [http://fs-demo.d8-3.local](http://fs-demo.d8-3.local)
```

Comment:
Thanks. I'm not sure where I came up with that scheme, but it's not really how anything is usually set up. I'm reorganizing, but wondering if I can just put the `cookies_domain` in `services.yml` for everything or if it will have any negative effects on the non-domain sites?

Reply:
Just did some reading and it seems like `cookie_domain` isn't going to hurt anything.

Reply:
Technically, it opens a potential security issue because you are sharing login cookies across sites, but this is on local so it doesn't matter.

If all the sites use `.fs-demo.local` as the root, it will be fine. However, if `cookie_domain` is not compatible with the hostname, then users won't be able to login.

Reply:
@agentrickard I've rejiggered all the things and also added a phing task to clear Solr, since all this rebuilding results in a lot of junk in the index. You'll need to do a full `vagrant halt && vagrant up --provision` to get the new hostnames but this should clean things up.

Reply:
I just pushed one new change to make the D8 Domain site use its own `services.yml` and now login seems to work consistently across the domain sites and also not knock out the standalone site in the process (it was breaking when the standalone site shared the same cookie, which makes sense).